src/app/page.tsx

```diff
@@ -1,6 +1,10 @@
+import Navbar from "@/components/Navbar";
+
 export default function Home() {
   return (
-    <main className='flex min-h-screen flex-col items-center justify-between p-24'>
+    <main className="flex min-h-screen flex-col items-center justify-between p-24">
+      <Navbar />
```

Comment:
this should go into the root layout if it's always visible (even for signed out users/guests) or on the dashboard one if it's only for users

Reply:
i added this here just so i could see it as we haven't defined anything yet